ietf/templates/submit/tool_instructions.html

```diff
@@ -30,15 +30,15 @@ <h2 class="mt-3">I-D submission tool instructions</h2>
     </p>
     <p>
         An API for automated Internet-Draft submission is available as an alternative to this webpage at
-        <a href="{{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.submit.views.api_submit' %}">{{ settings.IDTRACKER_BASE_URL }}{% url 'ietf.submit.views.api_submit' %}</a>.
+        <a href="https://datatracker.ietf.org/api/submission">https://datatracker.ietf.org/api/submission</a>.
```

Comment:
This should use the settings and url reversals. The view is `ietf.submit.views.api_submission`

Reply:
Sure.  That is fine with me.